console/style.rst

```diff
@@ -67,8 +67,8 @@ title of the command::
 
         protected function execute(InputInterface $input, OutputInterface $output)
         {
-            $io = new SymfonyStyle($input, $output);
-            $io->title('Lorem Ipsum Dolor Sit Amet');
+            $inputOutput = new SymfonyStyle($input, $output);
```

Comment:
$ioHelper would be more descriptive

Reply:
I like it, but Javier said:

>I think it's better to keep $io because we use it everywhere in the docs, the code, the Symfony Demo app, etc

tbh I did not find `$io` anywhere else, I will check again tonight. @javiereguiluz what do you think?

Reply:
Frankly it's one of the rare cases where I prefer the short name because it's such a common name even outside of Symfony. `$inputOutput` is _very_ verbose and a bit confusing with `$intput`, `$output` next to it. I could settle with  `$ioHelper` though